### ISSUE TITLE
Test failure - io.druid.server.lookup.PollingLookupTest.testApplyAfterDataChange

### ISSUE THREAD

**Issue by drcrallen** [apache/incubator-druid#3480]
As per https://github.com/druid-io/druid/pull/3476#issuecomment-248710267 the tests in `io.druid.server.lookup.PollingLookupTest` are randomly failing

**Comment by navis** [comment]
The message for me was different with  https://github.com/druid-io/druid/pull/3476#issuecomment-248710267

```
io.druid.server.lookup.PollingLookupTest
testApplyAfterDataChange[1](io.druid.server.lookup.PollingLookupTest)  Time elapsed: 0.456 sec  <<< ERROR!
java.lang.Exception: test timed out after 300 milliseconds
	at java.lang.Thread.sleep(Native Method)
	at io.druid.server.lookup.PollingLookupTest.testApplyAfterDataChange(PollingLookupTest.java:149)
```

**Comment by gianm** [comment]
Believed to be fixed by #3489.
